### PR TITLE
TS-4863: Remove non-existent proxy.node.num_processes metric.

### PR DESCRIPTION
**Pull request by jpeach**

### CODE REVIEW

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/699/ for details.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/804/ for details.

File: plugins/experimental/epic/epic.cc [hunk 1]
```diff
@@ -50,120 +50,192 @@
...
+  "proxy.process.http.user_agent_speed_bytes_per_sec_1M",
+  "proxy.process.log.log_files_open",
+  "proxy.process.log.log_files_space_used",
+  "proxy.process.net.accepts_currently_open",
+  "proxy.process.net.connections_currently_open",
+  "proxy.process.socks.connections_currently_open",
+  "proxy.process.update.state_machines",
+  "proxy.process.version.server.build_time",
+  "proxy.process.websocket.current_active_client_connections",
+};
```
Comment (shinrich) [inline plugins/experimental/epic/epic.cc]: Isn't this kind of an unrelated change?

Comment (jpeach) [inline plugins/experimental/epic/epic.cc]: > On Sep 13, 2016, at 5:34 PM, Susan Hinrichs <notifications@github.com> wrote:
> 
> In plugins/experimental/epic/epic.cc:
> 
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_100",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_100K",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_100M",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_10K",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_10M",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_1K",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_1M",
> > +  "proxy.process.log.log_files_open",
> > +  "proxy.process.log.log_files_space_used",
> > +  "proxy.process.net.accepts_currently_open",
> > +  "proxy.process.net.connections_currently_open",
> > +  "proxy.process.socks.connections_currently_open",
> > +  "proxy.process.update.state_machines",
> > +  "proxy.process.version.server.build_time",
> > +  "proxy.process.websocket.current_active_client_connections",
> > +};
> >  
> Isn't this kind of an unrelated change?
> 
Yeh I made it because the clang format made the original diff really big. I'll reorder the commits which should make the functional one much clearer.

**Comment by jpeach** [comment]
@shinrich There's still a gratuitous C++11 commit here, but in the second commit it is now very obvious what the change is.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/805/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/700/ for details.

File: plugins/experimental/epic/epic.cc (hunk 1 above)
Comment (shinrich) [inline plugins/experimental/epic/epic.cc]: I'm ok with the change in any event.  I see now the the num_process was in the gauge list which was why you were messing with it.  I didn't catch that the first time around, so the change in epic seemed completely unrelated.  But it is in fact related.
